src/guide/kotlin-java.md

```diff
@@ -77,24 +80,22 @@ and see how the `Iroha2Config` is implemented.
 
 <<<@/snippets/IrohaConfig.kotlin
 
-## 3. Registering a Domain
+## 3. Domain search and registration
 
-Registering a domain is one of the easier operations. The usual boilerplate
+Search Registering a domain is one of the easier operations. The usual boilerplate
```

Comment:
Why is the word search in this sentence and where is the prescription of the search function?

Reply:
update on hext commit